test/css/defaults.css

```diff
@@ -5826,13 +5826,16 @@ button.close {
  */
 @font-face {
   font-family: 'League Spartan';
-  src: url("../fonts/leaguespartan-bold.eot");
-  src: url("../fonts/leaguespartan-bold.eot?#iefix") format("embedded-opentype"), url("../fonts/leaguespartan-bold.woff2") format("woff2"), url("../fonts/leaguespartan-bold.woff") format("woff"), url("../fonts/leaguespartan-bold.ttf") format("truetype"), url("../fonts/leaguespartan-bold.svg#league_spartanbold") format("svg");
+  src: url("${ttn-font-path}leaguespartan-bold.eot");
```

Comment:
Changing `../fonts/` to `${ttn-font-path}` is not valid CSS and test fails with this change.